contents.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contents</title>
+</head>
+
+<body>
+    <h2>Table of Contents</h2>
```

Comment:
Please, create `div` wrapper with class it looks like `<div class="contents-wrapper">Your content in body tag</div>`